tests/aiohttp/flask_app/app1.py

```diff
@@ -50,8 +50,9 @@ async def fetch(url, body, headers):
     async with session.post(url, data=body, headers=headers) as response:
       return await response.text()
 
+
 @server.route("/route2", methods=['POST'])
-def testAPI1():
+def testAPI2():
```

Comment:
Please use descriptive names for tests e.g. what this test is trying to assert or test. A good example can be found here:

https://github.com/hypertrace/goagent/blob/main/instrumentation/opentelemetry/net/hyperhttp/handler_test.go#L22